test-asset-generator/transformers/transformer.js

```diff
@@ -44,13 +40,13 @@ export default class Transformer {
 
             switch (asset['@type']) {
```

Comment:
Using a switch could potentially lead to convoluted code later down the line. Plus it looks like all of the transformers do the same thing with structural output differences, it seems to me that instead of having three different transformer functions all doing something similar, it would be a lot more DRY if we had one transformer function with an 'asset_type' param in order to ensure the right output was given for each type.

Reply:
Probably just a software design discussion. In C# you can pass through interfaces and it auto-detects which class to use. I just didn't want everything in 1 mammoth file so this is better for readability.